cmscontrib/__init__.py

```diff
@@ -163,7 +165,8 @@ def _update_object(self, old_object, new_object, ignore=None):
 
             # General case #1: a dict
             elif isinstance(old_value, dict):
-                for key in set(old_value.keys()) | set(new_value.keys()):
+                for key in set(itertools.chain(iterkeys(old_value),
+                                               iterkeys(new_value))):
```

Comment:
Same here, but here it might be a bit faster to do it like you did (if there are repeated elements).

Reply:
I ran some tests and apparently the running time isn't influenced by how many keys the two dicts have in common. Yet, doing it with `set | set` is about 50% slower than doing it `set(chain)`. We're talking about ridiculously small values (2 and 3 microseconds for two dicts of 26 elements each) so I don't think performance is really relevant. For consistency with the above comment I'm going to revert to `set | set`.